tools/common_lib/src/tensor_shape.h

```diff
@@ -46,6 +46,11 @@ struct TensorShape
 
         return ret;
     }
+
+    inline std::uint32_t get_element_count() const
+    {
+        return (n * c * h * w);
```

Comment:
what if some dimension is equal to 0? please have a look at function above `get_dims_count()` and implement this in similar fashion

Reply:
Done